tests/run-make/allow-warnings-cmdline-stability/rmake.rs

```diff
@@ -0,0 +1,18 @@
+// Test that -A warnings makes the 'empty trait list for derive' warning go away
```

Comment:
Suggestion: Looking back at the original PR I think this test is actually more about testing that `-A warnings` still suppresses warnings for unstable APIs. Maybe we should explain that, because I think "empty trait list for derive" is just one particular thing used to test that intention.

Reply:
Makes more sense indeed. Updating the comment then.